v03_pipeline/lib/methods/sex_check.py

```diff
@@ -0,0 +1,49 @@
+import hail as hl
+
+from v03_pipeline.lib.model import Sex
+
+AMBIGUOUS_THRESHOLD_PERC: float = 0.01  # Fraction of samples identified as "ambiguous_sex" above which an error will be thrown.
+AAF_THRESHOLD: float = 0.05  # Alternate allele frequency threshold for `hl.impute_sex`.
+BIALLELIC: int = 2
+XX_FSTAT_THRESHOLD: float = (
+    0.5  # F-stat threshold below which a sample will be called XX
+)
+XY_FSTAT_THRESHOLD: float = (
+    0.75  # F-stat threshold above which a sample will be called XY.
+)
+
+
+def compute_sex_check_ht(mt: hl.MatrixTable) -> hl.Table:
+    # Filter to SNVs and biallelics
+    # NB: We should already have filtered biallelics, but just in case.
+    mt = mt.filter_rows(
+        (hl.len(mt.alleles) == BIALLELIC) & hl.is_snp(mt.alleles[0], mt.alleles[1]),
+    )
+
```

Comment:
this will NOT be perfect i don't think

```suggestion
mt = mt.filter_cols(
     hl.agg.all(
          mt.GT.is_diploid() | hl.is_missing(mt.GT)
          )
     )
)

```

Reply:
^ wouldn't this exclude samples that were not diploid?  wouldn't we want the callset to fail if there were inadvertent haploid calls?

Reply:
this would! which would just have excluded that one bad sample. 

Ben does this run on all samples in a callset, or after filtering to samples in the AirTable ?